### PR TITLE
Add plasma-panel-colorizer widget-specific options

### PR DESCRIPTION
**Pull request by HeitorAugustoLN**
I think this gets every option currently available, but I don't know, I will check it later.

### CODE REVIEW

**Comment by magnouvean** [comment]
Very nice. I think we'll wait until the package is in nixpkgs, and swiftly merge after some testing together with the other pr :)

File: modules/widgets/plasma-panel-colorizer.nix
Comment (magnouvean) [inline modules/widgets/plasma-panel-colorizer.nix]: I believe this line causes the build to fail atm. There is a missing `};` (which does mean that there is one too many `};` somewhere else probably.

Comment (HeitorAugustoLN) [reply modules/widgets/plasma-panel-colorizer.nix]: I didn't notice that. Will fix it.

Comment (HeitorAugustoLN) [reply modules/widgets/plasma-panel-colorizer.nix]: I haven't really tested these options yet, but I will do it soon.

**Comment by HeitorAugustoLN** [comment]
I think I will actually rewrite this.

**Comment by HeitorAugustoLN** [comment]
I just need some days to finish my config rewrite.

**Comment by HeitorAugustoLN** [comment]
Ok, this is fine now. I tested it a bit with these options:

image

Last time I couldn't test.

**Review by magnouvean — APPROVED** [review]
From the little I've tested as well this works great. Obviously haven't tested near all of the options but if there happens to be some non-working option we can easily fix that in a separate pr, so I think it's good for merging.

As always thanks a lot for the great PR! Oh, and if you ever want commit-access to the repo, it's just to let me know :)

**Comment by HeitorAugustoLN** [comment]
> Oh, and if you ever want commit-access to the repo, it's just to let me know :)

It would be really nice! Can I be also part of the plasma-manager team? I want to help you guys with maintaining this project.

**Comment by magnouvean** [comment]
Of course, I think your contributions are great :). Also if the changes aren't too major feel free to merge right away without waiting for my approval (I have a couple especially busy months now so I won't be as quick on PRs as I normally am).

@zowoq, could you add @HeitorAugustoLN to the @nix-community/plasma-manager team (it doesn't look like I have permissions to do so)?

**Comment by zowoq** [comment]
> could you add @HeitorAugustoLN to the @nix-community/plasma-manager team

Done.

> (it doesn't look like I have permissions to do so)?

If they aren't already a member of nix-community they first need to be invited by one of the organisation owners.

**Pull request merged**
